test/input.txt

```diff
@@ -147,6 +147,19 @@
   # clears all
   clear
   list
+  
+##########################################################
+# test new regex for name
+##########################################################	 
+  add Name with'arc p/99999999 e/name@arc.com a/1 long, #01-valid address, 890765
```

Comment:
You may want to add more test commands or modify this test here to cover additional details described by the new regex i.e. `.`, `-` and unicode letters.